attribute/genericattribute.go

```diff
@@ -0,0 +1,25 @@
+package attribute
+
+import (
+	"github.com/getyoti/yoti-go-sdk/anchor"
+)
+
+//GenericAttribute is a Yoti attribute which returns a generic value
+type GenericAttribute struct {
```

Comment:
Including the pkg name in the pkg member names results in stutter. How about just 'Generic', since the pkg name prefix lets us know its a attribute?